src/main/resources/assets/vivatech/lang/en_us.json

```diff
@@ -6,7 +6,17 @@
   "block.vivatech.energy_conduit": "Energy Conduit",
   "block.vivatech.coal_generator": "Coal Generator",
   "block.vivatech.crusher": "Crusher",
+  "block.vivatech.minimal_crusher": "Minimal Crusher",
+  "block.vivatech.normal_crusher": "Normal Crusher",
+  "block.vivatech.advanced_crusher": "Advanced Crusher",
   "block.vivatech.electric_furnace": "Electric Furnace",
+  "block.vivatech.minimal_electric_furnace": "Minimal Electric Furnace",
+  "block.vivatech.normal_electric_furnace": "Normal Electric Furnace",
+  "block.vivatech.advanced_electric_furnace": "Advanced Electric Furnace",
+  "block.vivatech.press": "Press",
+  "block.vivatech.minimal_press": "Minimal Press",
+  "block.vivatech.normal_press": "Normal Press",
+  "block.vivatech.advanced_press": "Advanced Press",
```

Comment:
Are the old translation keys (`block.vivatech.crusher` etc) needed anymore?

Reply:
I think it should be other way. It should not note the tier in the name. Tier should be in the Lore. You can do that via overriding block methods

Reply:
Fixed